__tests__/server.test.js

```diff
@@ -122,4 +123,54 @@ describe('GET /api/articles',()=>{
             expect(body.articles).toBeSortedBy('created_at',{descending:true})
         })
     })
+})
+
+describe('POST /api/articles/:article_id/comments',()=>{
+    test('should return a 201 status code with the posted comment',()=>{
+        return request(app)
+        .post('/api/articles/12/comments')
+        .send({username:'rogersop',body:'life changing'})
+        .expect(201)
+        .then(({body}) => {
+            const comment = body.comment
+            expect(typeof comment.comment_id).toBe('number')
+            expect(typeof comment.votes).toBe('number')
+            expect(typeof comment.created_at).toBe('string')
+            expect(typeof comment.author).toBe('string')
+            expect(typeof comment.body).toBe('string')
+            expect(typeof comment.article_id).toBe('number')
+        })
+    })
```

Comment:
Please add another test where extra properties of the post object are ignored.

For example when posting with this:
`{username:'rogersop',body:'life changing',extraProperty:'extraValue'}`

Reply:
I knew I was missing a test somewhere thanks!